### DISCUSSION TITLE
how can deploy medusa in vps server

### DISCUSSION THREAD

**Discussion by adilek96** [medusajs/medusa#9963] [Q&A]
don't judge strictly maybe my question will seem noobish, but after looking at all the available tutorials I still couldn't deploy medusa to my vps server, basically all the tutorials show how to deploy to railway, but I need to deploy to my server, docploy is installed on the server, I tried to do it with its help but nothing worked, please help me understand how to do this

**Comment by binarycombinatrixcoder** [comment]
Any updates on this one? I am trying to deploy solace medusa starte on a GCP VM

**Comment by Kyle-Lutz** [comment]
What part are you having trouble with? Im running medusa on a vps with digital ocean. I followed the general deployment guide. [https://docs.medusajs.com/learn/deployment/general](url)
